release-scripts/update-snapshot-version.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+VERSION=$1
+
+git checkout master
+
+mvn versions:set -DnewVersion=${VERSION}
+
```

Comment:
don't we need to actually commit to git here?

Reply:
Fixed!